### DISCUSSION TITLE
Adventurer neutral - allow skin color adjustments

### DISCUSSION THREAD

**Discussion by mortetm** [dicebear/dicebear#220] [Q&A] [answered]
I love the adventurer neutral avatars and would love if I could choose the skin color. I plan on using it for a DnD initiative tool I'm working on, but for PCs I have to be able to choose the skin color. Is there any chance for that to happen?

**Comment by FlorianKoerner** [comment, marked as answer]
Hi @mortetm

you can use the `backgroundColor` option to set the skin color.

**HTTP-API**

https://avatars.dicebear.com/api/adventurer-neutral/your-custom-seed.svg?backgroundColor=%23ff0000

**JS-Library**

```js
import { createAvatar } from '@dicebear/avatars';
import * as style from '@dicebear/adventurer-neutral';

const avatar = createAvatar(style, {
  seed: 'your-custom-seed',
  backgroundColor: ['#ff0000']
});
```

**Comment by mortetm** [comment]
Hey Florian! Thank you, that is great. Now I can have green-skinned orcs as well! This is an amazing project and I am so very happy to have found it.
